epm/Distribution/Client/IndexUtils.hs

```diff
@@ -450,7 +451,7 @@ packageIndexFromCache mkPkg hnd entrs mode = accum mempty [] entrs
       -- from the index tarball if it turns out that we need it.
       -- Most of the time we only need the package id.
       ~(pkg, pkgtxt) <- unsafeInterleaveIO $ do
-        mPatch <- patchedPackageCabalFile pkgid
+        mPatch <- patchedPackageCabalFile pkgid defaultPatchesDir
```

Comment:
On second thought, this is fine as it is now. We can ignore the 'Configure' built type if `epm` comes across it. I think that'll be a better solution for now.